signup/impl/src/bundle/emailMessage_es.properties

```diff
@@ -96,7 +96,7 @@ body.footer.text.no.access.link=Este mensaje de notificaci\u00f3n autom\u00e1tic
 ical.footer.text=Tenga en cuenta que rechazar la invitaci\u00f3n del calendario no le retira del evento. Por favor vuelva al sitio para hacer cualquier modificaci\u00f3n.
 ical.footer.separator=-------
 
-signup.event.attendees=Participantes:
-signup.event.attendeestitle={0} participantes
-signup.event.currentattendees=En este momento, {0} participantes se han inscrito en el evento.
+signup.event.attendees=Asistentes\:
+signup.event.attendeestitle={0} asistentes
+signup.event.currentattendees=Hasta la ahora se han inscrito {0} asistentes.
```

Comment:
Hasta ahora se han inscrito {0} asistentes.

Reply:
Thanks for the catch, Jesús. This property must be fixed in Transifex, the translation system that we (S2U) use. If you know somebody in UMU that has a Transifex account you can ask him to do it. Otherwise I will tell my workmate Pablo when he returns from his holidays.